brave/brave-core
# Hide query tracking parameter from Twitter Service Worker

Fixes brave/brave-browser#23742

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on
  - [ ] https://github.com/brave/brave-browser/wiki/Query-String-Filter

## Test Plan:

Renderer-initiated:

1. Open <https://twitter.com/brave> and leave open for 10 seconds so that the Service Worker is installed and active.
2. Open <https://fmarier.org/query-filter-test.html>.
3. Click on the last link on the page ("Twitter").
4. Check that the `fbclid=` parameter is missing from the URL bar.

Browser-initiated:

1. Open <https://twitter.com/brave> and leave open for 10 seconds so that the Service Worker is installed and active.
2. Paste <https://twitter.com/fmarier/status/1574869891978784769?s=21&t=lkfjeslkfjes&fbclid=1234> into the URL bar and press Enter.
3. Check that the `fbclid=` parameter is missing from the URL bar.
